source/cloud/aws/sagemaker.md

```diff
@@ -10,16 +10,15 @@ RAPIDS can be used in a few ways with [AWS SageMaker](https://aws.amazon.com/sag
 
 To get started head to [the SageMaker console](https://console.aws.amazon.com/sagemaker/) and create a [new SageMaker Notebook Instance](https://console.aws.amazon.com/sagemaker/home#/notebook-instances/create).
 
-Choose `Notebook > Notebook Instances > Create notebook instance`.
+Choose `Applications and IDEs > Notebooks > Create notebook instance`.
 
 ### Select your instance
 
 If a field is not mentioned below, leave the default values:
 
-- **NOTEBOOK_INSTANCE_NAME** = Name of the notebook instance
-- **NOTEBOOK_INSTANCE_TYPE** = Type of notebook instance. Select a RAPIDS-compatible GPU ([see the RAPIDS docs](https://docs.rapids.ai/install#system-req)) as the SageMaker Notebook instance type (e.g., `ml.p3.2xlarge`).
-- **PLATFORM_IDENTIFIER** = 'Amazon Linux 2, Jupyter Lab 3'
-- **IAM_ROLE** = Create a new role > Create role
+- **Notebook instance name** = Name of the notebook instance
+- **Notebook instance type** = Type of notebook instance. Select a RAPIDS-compatible GPU ([see the RAPIDS docs](https://docs.rapids.ai/install#system-req)) as the SageMaker Notebook instance type (e.g., `ml.p3.2xlarge`).
+- **Platform identifier** = 'Amazon Linux 2, Jupyter Lab 4'
```

Comment:
Changes here:

* use the same case as the UI
* remove "IAM Role" (these examples work with the default role added by SageMaker)
* update to `Amazon Linux 2, Jupyter Lab 4` (the latest platform SageMaker supports)